src/test/java/com/wikia/webdriver/elements/oasis/components/globalshortcuts/KeyboardShortcutsModal.java

```diff
@@ -0,0 +1,122 @@
+package com.wikia.webdriver.elements.oasis.components.globalshortcuts;
+
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.elemnt.Wait;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class KeyboardShortcutsModal {
+
+  @FindBy(css =".global-shortcuts-help .close")
+  private WebElement closeButton;
+
+  @FindBy(css = ".global-navigation .search-input")
+  private WebElement globalNavigationSearchInput;
+
+  private By keyboardShortcutsModalSelector = By.cssSelector(".global-shortcuts-help");
+  private By actionExplorerModalSelector = By.cssSelector(".global-shortcuts-search");
+
+  private Wait wait;
+  private Actions actions;
+  private WebDriver driver;
+
+  private static final String INSIGHTS_PAGE = "/wiki/Special:Insights";
+
+  public KeyboardShortcutsModal(WebDriver driver) {
+    this.wait = new Wait(driver);
+    this.actions = new Actions(driver);
+    this.driver = driver;
+
+    PageFactory.initElements(driver, this);
+  }
+
+  public KeyboardShortcutsModal clickCloseButton() {
+    wait.forElementClickable(closeButton);
+    closeButton.click();
+
+    wait.forElementNotVisible(keyboardShortcutsModalSelector);
+    PageObjectLogging.logInfo("Keyboard shortcuts modal was closed by close button");
+
+    return this;
+  }
+
+  public KeyboardShortcutsModal useShortcut(String shortcut) {
+    switch (shortcut) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a default case to this switch. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3ASwitchLastCaseIsDefaultCheck)